artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ServerLocatorImpl.java

```diff
@@ -94,16 +94,11 @@ private enum STATE {
       FluentPropertyBeanIntrospectorWithIgnores.addIgnore(ServerLocatorImpl.class.getName(), "setThreadPools");
    }
 
-   private static final long serialVersionUID = -1615857864410205260L;
-
    // This is the default value
    private ClientProtocolManagerFactory protocolManagerFactory = new ActiveMQClientProtocolManagerFactory().setLocator(this);
 
    private final boolean ha;
 
-   // this is not used... I'm only keeping it here because of Serialization compatibility and Wildfly usage on JNDI.
-   private boolean finalizeCheck;
```

Comment:
This one is strange since, it was only made this way in recent years, but the class wasnt Serializable at the time. The history of the file is littered with references to serialization...but the file was seemingly never Serializable in the time since Artemis existed. There is a nested class in the file that _is_ Serializable though.

Is there perhaps another type of serializing the locator that the earlier change here thought needed to be concious of not breaking?

Reply:
I just left this one alone.